stuff/pipelines/client_contest.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
+
+'''Processing contest messages.'''
+
+import argparse
+import logging
+import os
+import sys
+import json
+from typing import Any, List, Optional, Tuple
+import MySQLdb
+import MySQLdb.cursors
+import pika
+from verification_code import generate_code
+import rabbitmq_connection
+
+import omegaup.api   # pylint: disable=import-error
```

Comment:
```suggestion
import omegaup.api
```

hay que evitar suprimir errores antes de mandarlo a GitHub. así podemos ver qué es lo que falla.

Reply:
Listo. Ya lo quité.